### PR TITLE
issue #977 remove derby config from bulkdata.xml

### PR DESCRIPTION
**Pull request by albertwang-ibm**
Signed-off-by: Albert Wang <xuwang@us.ibm.com>

### CODE REVIEW

File: fhir-server/liberty-config/configDropins/defaults/bulkdata.xml [hunk 1]
```diff
@@ -41,15 +44,21 @@
...
             serverName="localhost"
             user="javabatch"
-            password="change-password" /> -->
-        <properties.derby.embedded createDatabase="create" databaseName="derby/fhirDB"/>
+            password="change-password" />
     </dataSource>
+    -->
 
     <batchPersistence jobStoreRef="BatchDatabaseStore"/>
     <databaseStore dataSourceRef="fhirbatchDS" id="BatchDatabaseStore" schema="JBATCH" tablePrefix=""/>
```
Comment (lmsurpre) [inline fhir-server/liberty-config/configDropins/defaults/bulkdata.xml:64]: so it doesn't complain that `fhirbatchDS` doesn't exist by default?

Comment (albertwang-ibm) [reply fhir-server/liberty-config/configDropins/defaults/bulkdata.xml:64]: yes, it doesn't complaint on these 2 lines, so I kept them here, and so only need to define the datasource in the overrides

Comment (albertwang-ibm) [reply fhir-server/liberty-config/configDropins/defaults/bulkdata.xml:64]: and yes, I verified that derby is used in this case. the job id is always 1 after each restart of the server.

Comment (lmsurpre) [reply fhir-server/liberty-config/configDropins/defaults/bulkdata.xml:64]: OK, I havn't quite got around to test bulk export locally, but I'll take your word for it :-)

Comment (albertwang-ibm) [reply fhir-server/liberty-config/configDropins/defaults/bulkdata.xml:64]: thanks! :)

**Pull request merged**